### ISSUE TITLE
Connection with  "PDO::FETCH_ASSOC" option.

### ISSUE THREAD

**Issue by DaaGER** [usmanhalalit/pixie#143]
Sorry, for my english.

Hi.
I want "array", instead of "object" in all queries.
My config:

```
$config = array(
        'driver'   => 'mysql',
        'host'     => $db_conf['host'],
        'database' => $db_conf['dbname'],
        'username' => $db_conf['user'],
        'password' => $db_conf['pass'],
        'charset'  => 'cp1251',
        'options'=>[\PDO::ATTR_DEFAULT_FETCH_MODE=>\PDO::FETCH_ASSOC]
or

        'options'=>[PDO::FETCH_ASSOC=>true]
    );
```

And this case don't work, but if I using "setFetchMode" it's working.
Everytime write "setFetchMode" I dont want. Please, help.

**Comment by eL-Prova** [comment]
Hi,

I have no idea what you want.
Can you make yourself clear with an example?

**Comment by DaaGER** [comment]
I want know, how i can get result of quries in array, without use "setFetchMode" everytime.

**Comment by eL-Prova** [comment]
Ok you want an array instead of a stdObject
You can use events to make that possible. You register on each select query after select that the fetchmode will be translated to an array.

In your event you are using this:

$array = json_decode(json_encode($object), True);

**Comment by TCB13** [comment]
@eL-Prova @DaaGER I know this an old issue, however the same request surfaced again at #185. I believe that you were misunderstood at the time @DaaGER .

**Comment by TCB13** [comment]
Fixed by 220b920904b9f60a7d61eba9554d936f4fda7977
